Packs/PrismaCloud/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Prisma Cloud",
```

Comment:
make sure the current pack metadata contains certification status, tags, price, usecases, etc
https://xsoar.pan.dev/docs/integrations/pack-docs#pack-description
https://xsoar.pan.dev/docs/integrations/packs-format#pack_metadatajson

Reply:
updated.